app/components/views/TicketsPage/MyTicketsTab/Page.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
```

Comment:
`React` is being provided by webpack's providePlugin.  No need to include this anymore.